tests/e2e/testRunner.js

```diff
@@ -115,6 +117,7 @@ const runTestsOnBranch = async (baselineOrCompare, branch) => {
         const tempDir = `${config.OUTPUT_DIR}/temp`;
         const tempBundlePath = `${tempDir}/index.android.bundle`;
         await execAsync(`rm -rf ${tempDir} && mkdir ${tempDir}`);
+        // TODO: test if that breaks support for the env file
```

Comment:
@AndrewGable I think we can get rid of this TODO comment as the AWS tests have passed, which means that everything is working alright